docs/api/api/configuration.rng

```diff
@@ -198,6 +198,18 @@
             </choice>
           </element>
         </optional>
+        <optional>
+          <element name="excluded_projects">
+            <!-- The exclusion filter itself. Is a Regular Expression (eg: ^home:.*) -->
+            <text/>
+          </element>
+        </optional>
+        <optional>
+          <element name="excluded_projects_description">
```

Comment:
Maybe `projects_exclusion_regex_description`? Or  `projects_exclusion_description`? Just to keep it consistent with the previous config var..